alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/healthprocessor/plugins/solr/SolrRequestExecutor.java

```diff
@@ -151,14 +165,17 @@ private SolrActionResponse parseActionWebResponse(JsonNode response, String core
         private final String message;
     }
 
-
     @AllArgsConstructor
     public enum SolrNodeCommand {
-        REINDEX("reindex"),
-        PURGE("purge");
+        REINDEX("reindex", false),
+        PURGE("purge", false),
+        REINDEX_TRANSACTION("reindex", true),
+        PURGE_TRANSACTION("purge", true);
```

Comment:
This `PURGE_TRANSACTION` command is not used anywhere, so it should be removed completely.